utils.groovy

```diff
@@ -1,52 +1,9 @@
-import org.yaml.snakeyaml.Yaml;
+@Library('github.com/coreos/coreos-ci-lib@main') _
```

Comment:
I'm not 100% sure if this will work from within a separate module like this. But we can try it out and at worst, we'll just use `sh` directly in those few places for now.